### PR TITLE
Chima: Setup storybook

### PR DESCRIPTION
**Pull request by chukwuemekachm**
#### What does this PR achieve?
- This PR introduces the required changes to setup `storybook` on the codebase for documenting components. It also writes `stories` for existing components.

#### Any background context?
- We need a way of visually documenting the components on our codebase. `storybook` offers a great solution for visually documenting UI components built with React JS.
- [Storybook](https://storybook.js.org/)

#### Manual Testing Steps
- Run `yarn` to install dependencies
- Run `yarn storybook` to start `storybook`

#### Screenshots:

Screenshot 2019-12-28 at 9 47 52 PM

Screenshot 2019-12-28 at 9 47 34 PM

#### Relevant Trello Tickets
- [#43](https://trello.com/c/jKHCeLQF)

### CODE REVIEW

File: src/components/layouts/Logo/index.tsx [hunk 1]
```diff
@@ -3,19 +3,19 @@ import styled from '@emotion/styled';
...
+import { fontWeights, fontSizes } from 'settings/__fonts';
 import __layouts from 'settings/__layouts';
 
 interface LogoProps {
-    center?: boolean,
+  hasText?: boolean,
 }
 
 
-export default function Logo({center=false}: LogoProps) {
```
Comment (chidioguejiofor) [inline src/components/layouts/Logo/index.tsx:14]: Not sure why you removed this but removing this `center` prop from the logo actually breaks the ConfirmPassword page.

Screenshot 2019-12-28 at 10 08 51 PM

File: src/components/layouts/Logo/index.tsx [hunk 2]
```diff
@@ -24,11 +24,11 @@ Logo.Wrapper = styled.h2<LogoProps>`
...
   align-items: baseline;
   font-weight: ${fontWeights.normal};
-  margin-left: ${props=> props.center? '40%' : 0};
+  margin: 0;
 
   i { 
-    width: 45px;
-    height: 45px;
+    width: ${fontSizes.xLarge};
+    height: ${fontSizes.xLarge};
```
Comment (chidioguejiofor) [inline src/components/layouts/Logo/index.tsx:31]: Nice 👍

File: src/components/pages/Authentication/SignUp/SignUpForm.tsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ function SignUpForm(props: AuthenticationProps) {
         value={values.confirmPassword}
         errorFeedback={errors.confirmPassword}
       />
-    <Button type="submit" value='Sign Up'  isLoading={isLoading} />
+    <Button type="submit" isLoading={isLoading}>Sign Up</Button>
```
Comment (chidioguejiofor) [inline src/components/pages/Authentication/SignUp/SignUpForm.tsx:65]: Nice

File: src/components/ui/EmailSentSuccess/index.tsx [hunk 4]
```diff
@@ -11,7 +11,7 @@ const EMAIL_IMAGE = "https://trello-attachments.s3.amazonaws.com/5db1b02b5b9a3e4
 function EmailSentSuccess({ message }) {
   return (
     <EmailSentSuccess.Wrapper>
-        <EmailSentSuccess.Image />
+        <EmailSentSuccess.Image src={EMAIL_IMAGE} alt="Mail Sent Image" />
```
Comment (chidioguejiofor) [inline src/components/ui/EmailSentSuccess/index.tsx:14]: 👍

File: src/components/ui/Input/index.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+import Input from '.';
+
+// export default {
+//   component: Input,
+//   title: 'components/ui/Input',
+// };
+
+// export const largeLoader = () => (
+//   <Input size="large" />
+// );
```
Comment (chidioguejiofor) [inline src/components/ui/Input/index.stories.tsx:11]: I think you forgot to clean this

**Review by chidioguejiofor — COMMENTED** [review]
Looks great. Dropped a few feedback that could improve it

File: src/components/layouts/Logo/index.tsx (hunk 1 above)
Comment (chukwuemekachm) [reply src/components/layouts/Logo/index.tsx:14]: Is this for mobile or desktop view? Why is the overlay image absent?
I think this is not in line with the designs. Can you look at them again on Figma? All the Authentication pages are alike.

File: src/components/ui/Input/index.stories.tsx (hunk 5 above)
Comment (chukwuemekachm) [reply src/components/ui/Input/index.stories.tsx:11]: Thanks for the catch. I was going to write the story for that.

File: src/components/layouts/Logo/index.tsx (hunk 1 above)
Comment (chidioguejiofor) [reply src/components/layouts/Logo/index.tsx:14]: Just saw it now. Guess we would have to raise a bug fix for that one

**Review by chidioguejiofor — APPROVED** [review]
@chukwuemekachm  I think we can merge this in but we would need to raise a couple of fixes namely for the `ConfirmPassword`  page and the `Input` story book

**Comment by chukwuemekachm** [comment]
> @chukwuemekachm I think we can merge this in but we would need to raise a couple of fixes namely for the `ConfirmPassword` page and the `Input` story book

Okay, that's fine.

**Pull request merged**